# Translation of clouderp Server.
# This file contains the translation of the following modules:
# * im_clouderp_support
# 
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: clouderp 9.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-09-29 10:26+0000\n"
"PO-Revision-Date: 2015-09-30 09:26+0000\n"
"Last-Translator: Martin Trigaux\n"
"Language-Team: Portuguese (Brazil) (http://www.transifex.com/clouderp/clouderp-9/language/pt_BR/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: pt_BR\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#. module: im_clouderp_support
#. gce-web
#: code:addons/im_clouderp_support/static/src/js/im_clouderp_support.js:76
#, python-format
msgid "Error"
msgstr "Erro"

#. module: im_clouderp_support
#. gce-web
#: code:addons/im_clouderp_support/static/src/js/im_clouderp_support.js:76
#, python-format
msgid ""
"The connection with the clouderp Support Server failed. Please retry in a few "
"minutes, or send an email to support@golive.pt ."
msgstr "A conex√£o falhou. Por favor, tente novamente em alguns minutos , ou envie um e-mail para support@golive.pt"
